.github/workflows/integration-test-helm-suite.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-versions: ["v1.27.16", "v1.31.0"]
+        kubernetes-versions: ["v1.27.16", "v1.32.0"]
```

Comment:
Should we consider bumping up 1.27 to 1.28 also?

Reply:
We just bumped the min version to 1.27 in #15057 a couple weeks ago in anticipation of the release. Now with 1.32 we are at the goal of the last [six version of K8s supported](https://rook.io/docs/rook/latest-release/Getting-Started/release-cycle/#k8s-versions).